apps/site/lib/site_web/templates/stop/_detailed_stop.html.eex

```diff
@@ -4,7 +4,9 @@
     <%= if assigns[:include_route_icon?], do: stop_feature_icon(@detailed_stop.route_icon, :small) %>
     <%= feature_icons(@detailed_stop)%>
   </div>
-  <div class="m-detailed-stop__zone">
+  <%# For now we will not be showing fare zones since it doesn't help riders %>
+  <%# (reusing class .m-trip-plan__hidden for hiding) %>
+  <div class="m-trip-plan__hidden m-detailed-stop__zone">
```

Comment:
Bootstrap comes with utility classes, I think `hidden-xs-up` will hide it here.
(Admittedly the Bootstrap documentation for our version (v4.0.0-alpha.2) is hard to find, but the source code is [here](https://github.com/twbs/bootstrap/tree/v4.0.0-alpha.2/scss))